views/build/grunt/portableelement.js

```diff
@@ -219,7 +220,9 @@ module.exports = function (grunt) {
             grunt.log.writeln(`Only searching portable element "${selectedId}"`);
         }
 
-        manifests = portableModels.reduce(
+        models = type ? portableModels.filter(model => model.type === type) : portableModels;
```

Comment:
**suggestion:** Add error handling in case the type mismatches

Reply:
Added in https://github.com/oat-sa/extension-tao-itemqti/pull/2456/commits/d85de92d2c25ddd32d4d3060a2e40ece7eb6e0fe.